lib/Bootstrap.php

```diff
@@ -19,7 +19,9 @@
 use Pimcore\Model\DataObject;
 use Pimcore\Model\Document;
 use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\ErrorHandler\BufferingLogger;
 use Symfony\Component\ErrorHandler\Debug;
+use Symfony\Component\ErrorHandler\ErrorHandler;
```

Comment:
Hi @robertSt7 I guess you can also remove these use statements again since they are not used anymore.

Reply:
Why didn't the build catch these? :thinking: